overview.tt

```diff
@@ -0,0 +1,279 @@
+        own way of building and testing. Encourage cross-team development by
+        providing a single way of building everything.
+      </p>
+      <ul>
+        <li>language agnostic</li>
+        <li>portable</li>
+      </ul>
+      <div>
+        <p>
+          Keeping the project's installation instructions up to date is a never
+          ending task. Nix allows you to specify your system dependencies
+          alongside your project's dependencies in a declarative way.
+        </p>
+        <p>
+          Instead of complicated installation instructions with Nix the
+          development environment is always only one command away. And that is
+          without using any Docker or other vitalization.
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-looking.svg" %]
+    <div>
+      <h2>
+        Either it works or fails always in the same way.
+      </h2>
+      <p>
+        With Nix there are two results: the code either works or fails always
+        in the same way. This gives you the power to reproduce (and avoid) any
+        build related bugs .
+      </p>
+      <ul>
+        <li>reproducibility</li>
+        <li>transparent source binary model</li>
+        <li>nix language</li>
+        <li>continuous integration</li>
+      </ul>
+      <div>
+        <p>
+          Trying to keep system dependencies of development environments in
+          sync with continuous integration (CI) is a disaster waiting to
+          happen. Being able to use the same environment locally or anywhere
+          else is putting a developer back in control.
+        </p>
+        <p>
+          Build instructions are written in Nix language, which allows you to
+          extend or change existing build instruction in any way you wish. This
+          makes Nix a powerful build tool that can be used in both simple and
+          complex projects.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div>
+    [% PROCESS svg path="site-styles/assets/gfx-overview-cloud.svg" %]
+    <div>
+      <h2>
+        From zero to cloud.
+      </h2>
+      <ul>
+        <li>cloud images</li>
+        <li>docker images</li>
+        <li>devops</li>
+      </ul>
+      <p>
+        Use the same tool to build one project or even one hundred projects,
+        configure machines and deploy to the cloud. Thanks to the extensible
+        Nix language, it's easy to compose parts together with little overhead.
+      </p>
+      <div>
+        <p>
+          Use one tool to develop, test and build Amazon EC2, Google Cloud,
+          Azure, Virtualbox, KVM and other images in a declarative way is a
+          DevOps dream.
+        </p>
+        <p>
+          Forget <code>Dockerfile</code> and
+          have a peace of mind with declarative approach Nix gives you. Nix
+          automatically calculates the optimal layer layout and only includes
```

Comment:
"optimal layer layout" is getting pretty technical so we may want to avoid it here.